File: exs/mundo_1/java/006.java
/*
 * Desafio 006
 * problema: Crie um algoritmo que leia um número e mostre o seu dobro, o seu 
 * triplo e sua raiz quadrada.
 *
 * Resolução:
*/
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.printf("Número: ");
        int num = scan.nextInt();
        System.out.printf("Dobro = %d\n", num*2);
        System.out.printf("Triplo = %d\n", num*3);
        System.out.printf("Square root = %f\n", Math.sqrt(num));
    }
}

File: exs/mundo_1/c/006.c
/*
    006 - Dobro, triplo e raiz quadrada
        Crie um algoritmo que leia um número e mostre o seu dobro, o seu triplo e sua
        raiz quadrada
*/

#include <stdio.h>
#include <math.h>

int main(void)
{   
    int num;
    printf("Digite um número: ");
    scanf("%i", &num);

    printf("O dobro de %i é %i\n", num, num * 2);
    printf("O triplo de %i é %i\n", num, num * 3);
    printf("A raiz quadrada de %i é %.2f\n", num, sqrt(num));

    return 0;
}